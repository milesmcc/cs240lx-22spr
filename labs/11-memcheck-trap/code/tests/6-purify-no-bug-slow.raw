opened tty port </dev/cu.usbserial-0001>.
my-install: tty-usb=</dev/cu.usbserial-0001> program=<./tests/6-purify-no-bug-slow.bin> about to boot
TRACE:simple_boot: sending 30020 bytes, crc32=1b6df659
waiting for a start...
PRINT_STRING:pi sent print: <glen3b: success: Received the program!>
bootloader: Done.
TRACE:should have no error: running with verbose off
TRACE:ERROR:Allocated block 1 [0x183bc8] corrupted at offset -4
TRACE:	logical block id=1, [addr=0x183bc8] nbytes=824
TRACE:	Block allocated at: tests/6-purify-no-bug-slow.c:notmain:14
TRACE:ERROR:Allocated block 1 [0x183bc8] corrupted at offset -3
TRACE:	logical block id=1, [addr=0x183bc8] nbytes=824
TRACE:	Block allocated at: tests/6-purify-no-bug-slow.c:notmain:14
PANIC:../../labs/9-memcheck-stat/code.staff/ckalloc.c:ckalloc:150:mem_check(h, ck_get_rz1(h), rn)
DONE!!!

bootloader: pi exited.  cleaning up
