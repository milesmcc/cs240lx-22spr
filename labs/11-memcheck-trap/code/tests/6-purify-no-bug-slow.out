TRACE:should have no error: running with verbose off
TRACE:ERROR:Allocated block 1  corrupted at offset -4
TRACE:	logical block id=1,  nbytes=824
TRACE:	Block allocated at: tests/6-purify-no-bug-slow.c:notmain:14
TRACE:ERROR:Allocated block 1  corrupted at offset -3
TRACE:	logical block id=1,  nbytes=824
TRACE:	Block allocated at: tests/6-purify-no-bug-slow.c:notmain:14
PANIC:../../labs/9-memcheck-stat/code.staff/ckalloc.c:ckalloc:150:mem_check(h, ck_get_rz1(h), rn)
